pkgs/applications/misc/rtfm/gtk-docs.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv
+, fetchFromGitHub
+, gi-docgen
+, gobject-introspection
+, libadwaita
+, gtk4
+, pango
+}:
+stdenv.mkDerivation rec {
```

Comment:
Use the `finalAttrs` pattern please

Reply:
I did not quite understand how `finalAttrs` work and what it is used for.

Reply:
Let me share a couple of useful links regarding this:

History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293

Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes

Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Reply:
Thank you so much. There are many contentious issues when it comes to packaging. At the beginnings it is not easy to understand, i hope i did it right.

Reply:
I'll review it when I'm back on a decent laptop